contents/articles/2017-03_hololens-bt300-addon/index.md

```diff
@@ -0,0 +1,60 @@
+--- 
+title: Hololens and BT300 VR/AR Addon
+date: Thu Mar 02 2017 15:28:39 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "We are happy to announce that we have added the Microsoft Hololens and Epson Moverio BT-300 VR/AR addons, now available on the store!"
```

Comment:
We are excited to announce new AR eye tracking add-ons for Microsoft Hololens and the Epson Moverio BT-300 - now available on the store!